### ISSUE TITLE
Lotus journal files size growing very fast

### ISSUE THREAD

**Issue by egoleecode** [filecoin-project/lotus#8367]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [X] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
lotus version 1.15.0+mainnet+git.0ac1bbc
```

### Describe the Bug

I am testing the lotus storage transaction, I initiated dozens of storage transactions on my lotus lite node, but after a few days, I found that the file under the ~/.lotus/journal/ path is huge, almost occupying all my disks Space (I found that journal files are growing at 800KiB/s). I have adjusted several settings related to the log level but have not found an effective solution. If this problem persists, it will bring great inconvenience to the stable use of Lotus in the production environment.

### Logging Information

```text
Sample file in ~/.lotus/journal/
https://gateway.ipns.tech/ipfs/QmX4AoXpz9PoMDeBpEJiYrXvhEgS1HE3eU2nDRyGT9AEdd/lotus-journal.ndjson
```

**Comment by TippyFlitsUK** [comment]
Hi @egoleecode 

Thanks for the report. I will add labels and will assign it to the right team for analysis.
We might tag you again for more information.

**Comment by TippyFlitsUK** [comment]
Hi @egoleecode 

This issue has been resolved in [1.15.2-rc1](https://github.com/filecoin-project/lotus/releases). Please update this ticket if you continue to encounter this issue.

Many thanks!
